packages/datadog_flutter_plugin/test/datadog_sdk_test.dart

```diff
@@ -268,6 +268,28 @@ void main() {
         logger!.loggerHandle, loggingConfiguration));
   });
 
+  test('initialize with logging sample rate creates logger with sample rate',
```

Comment:
It seems that this test is identical to the test above, I don't see any setup/assert for the sample rate here.

Reply:
Ah, good call. I think this is left over from when I put the sample rate on the global configuration and it never got removed.